package.json

```diff
@@ -9,15 +9,17 @@
 		"biome:fix": "biome check . --write",
 		"//====== PREPARE ======//": "",
 		"prepare": "husky install",
-		"//====== LAUNCHING SCRIPTS ======//": "",
-		"storybook": "pnpm --filter @oscd-plugins/ui storybook",
 		"//====== LOCAL DEPENDENCIES INSTALL ======//": "",
 		"install:core": "pnpm --filter @oscd-plugins/core i",
 		"install:ui": "pnpm --filter @oscd-plugins/ui i",
+		"install:dependencies": "pnpm install:core && pnpm install:ui",
+		"//====== STORYBOOK ======//": "",
+		"storybook": "pnpm install:ui && pnpm --filter @oscd-plugins/ui storybook",
 		"//====== PLUGIN INSTALL ======//": "",
-		"install:type-designer": "pnpm install:core && pnpm install:ui && pnpm --filter @oscd-plugins/type-designer i",
+		"install:type-designer": "pnpm install:dependencies && pnpm --filter @oscd-plugins/type-designer i",
 		"//====== PLUGIN STAND ALONE MODE ======//": "",
-		"dev:type-designer": "pnpm install:type-designer && pnpm --filter @oscd-plugins/type-designer dev",
+		"dev:type-designer": "pnpm --filter @oscd-plugins/type-designer dev",
+		"dev:type-designer:install": "pnpm install:type-designer && pnpm --filter @oscd-plugins/type-designer dev",
```

Comment:
What's the idea for splitting install & "non install" scripts? How about whenever we run dev it automatically installs the dependecies as well?

Reply:
My original attempt was to have a flag that installs the dependencies on the first launch in a same terminal session. 
I fire this multiple times a day without having to install everything, everytime.

But I can't make it work (export variables from within the package.json scripts seems to be scoped to its command), that's why two commands (one for instantiation, the other for running the project faster)